imdclient/IMDClient.py

```diff
@@ -237,6 +249,17 @@ def _go(self):
         self._conn.sendall(go)
         logger.debug("IMDClient: Sent go packet to server")
 
+    def _wait(self, IMDwait=None):
```

Comment:
This method looks great, though one nitpick- capitalizing like this is usually reserved for class names, so IMDwait should be lowercase

Reply:
Also, I think this should be a method of the IMDv3Producer, not the IMDClient class since the IMDClient can also handle IMDv2 streams